### PR TITLE
Update preview2 branding

### PR DESCRIPTION
**Pull request by livarcocc**
@eerhardt @piotrpMSFT @brthor @jgoshi @leecow

### CODE REVIEW

File: branchinfo.txt [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # Each line is expected to be in the format "[Name]=[Value]".
 MAJOR_VERSION=1
 MINOR_VERSION=0
-PATCH_VERSION=0
+PATCH_VERSION=2
```
Comment (eerhardt) [inline branchinfo.txt]: This isn't right.  These are the CLI's version numbers.  We haven't shipped a CLI `1.0.0` yet, so it shouldn't be `1.0.2`.

File: packaging/windows/clisdk/registrykeys.wxs [hunk 2]
```diff
@@ -10,7 +10,7 @@
       </Component>
       <Component Id="SetupRegistry_x86_RC2_Compat_Key" Directory="TARGETDIR" Win64="no">
         <RegistryKey Root="HKLM" Key="SOFTWARE\dotnet\Setup\InstalledVersions\$(var.Platform)\sdk">
-          <RegistryValue Action="write" Name="1.0.0-preview2" Type="integer" Value="1" KeyPath="yes"/>
+          <RegistryValue Action="write" Name="1.0.2-preview2" Type="integer" Value="1" KeyPath="yes"/>
```
Comment (leecow) [inline packaging/windows/clisdk/registrykeys.wxs]: @livarcocc - Isn't this the SDK version string? If so, it should remain 1.0.0-preview2.

File: branchinfo.txt (hunk 1 above)
Comment (livarcocc) [inline branchinfo.txt]: Then we shouldn't want the installer to be dotnet-dev-osx-x64.1.0.2-preview2-buildnumber. @leecow , can you comment on it?

Comment (livarcocc) [inline branchinfo.txt]: Chatted with Lee, you are right Eric. Fixing.

**Comment by livarcocc** [comment]
@dotnet-bot Test CentOS7.1 x64 Debug Build

**Comment by livarcocc** [comment]
@dotnet-bot Test CentOS7.1 x64 Debug Build

**Pull request merged**